src/builders_v2/linux_vm/extlinux.rs

```diff
@@ -0,0 +1,175 @@
+use anyhow::{anyhow, Context, Result};
+use log::{debug, info};
+use std::ffi::OsStr;
+use std::fs;
+use std::io::Write;
+use std::path::{self, Path, PathBuf};
+
+use crate::builders::Step;
+
+use super::mbr::Mbr;
+use super::utils::run_command;
+use super::{InitSystemOpts, LinuxVMBuildContext};
+
+const DEFAULT_EXTLINUX_DIR: &str = "/boot/extlinux";
+
+/// This boot code is copied from extlinux 6.04.
+const BOOTCODE: &[u8; 440] = include_bytes!(concat!(
+    env!("CARGO_MANIFEST_DIR"),
+    "/src/builders_v2/linux_vm/data/extlinux/mbr.bin"
+));
+
```

Comment:
Hmm... This is slightly in grey area. After a quick peek, it seems that the syslinux/extlinux and it's mbr are under GPL. When embedding pre-compiled binary _inside_ a binary, does it count as redistribution or linking (legally)? At least the copyright and license should probably be present somewhere.

Reply:
Tbh I have absolutely no idea what we should do here :) Does it mean that we have to include GPL to our repo?

Reply:
This include is removed in the following commits.